### ISSUE TITLE
'Find in Files' state can become stale if project relocated

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#3053]
### System details

    RStudio Edition : Desktop
    RStudio Version : v1.2.xyz
    OS Version      : macOS 10.13.5
    R Version       : 3.5.0

### Steps to reproduce the problem

Create a project, execute a Find in Files action (<kbd>Ctrl + Shift + F</kbd>), and then try renaming or moving that project to a new location, re-open that project, and attempt to Find in Files again.

### Describe the problem in detail

The old project directory is saved and used.

### Describe the behavior you expected

The active project directory should be used.

### Notes

https://community.rstudio.com/t/find-in-files-always-finds-nothing-in-package-development-mode/10056/4

We save some persistent state about the Find in Files dialog as e.g.:

```r
7733176B/pcs/find-in-files.pper
6:        "path" : "~/scratch/project-path",
```

Note that the path is saved as a 'full' path; we should just save a project-relative path (for when the Find in Files path is indeed within the project) and then re-resolve the path.

**Comment by dfalty** [comment]
Verified fixed on version 1.2.1090.

**Comment by dmenne** [comment]
Looks like not fixed, 1.4.163. I had renamed the project, and nothing was found, even if everything seems to work. Deleting Rproj. helped.
